src/styles/token.ts

```diff
@@ -1,6 +1,39 @@
 const DESIGN_TOKEN = {
   color: {
-    primary: '#ff9980',
+    primary: {
+      p1: '#070220',
+      p2: '#120554',
+      p3: '#1E0887',
+      p4: '#290CBA',
+      p5: '#340FED',
+      p6: '#E8E6F3',
+    },
+    secondary: {
+      p1: '#E89582',
+      p2: '#E86F54',
+      p3: '#E94925',
+      p4: '#E8360E',
+      p5: '#4D180C',
```

Comment:
why use p prefix?

Reply:
Use 1-6 or 100-600 instead.

Reply:
when i use 1-6 or 100-600 i don't know why it error when i use in color tag such as
 palette: {
    primary: {
      main: DESIGN_TOKEN.color.primary.3,
                                                              ~~
    },

Reply:
fill in [] ->  DESIGN_TOKEN.color.primary[3]